src/main/java/com/volasoftware/tinder/mapper/FriendMapper.java

```diff
@@ -0,0 +1,19 @@
+package com.volasoftware.tinder.mapper;
+
+import com.volasoftware.tinder.dtos.FriendDto;
+import com.volasoftware.tinder.models.Account;
+import java.util.ArrayList;
+import java.util.List;
+import org.mapstruct.InheritInverseConfiguration;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.ReportingPolicy;
+
+@Mapper(componentModel = "spring", uses = LocationMapper.class, unmappedTargetPolicy = ReportingPolicy.IGNORE)
+public interface FriendMapper {
+
+    @Mapping(source = "location", target = "locationDto")
```

Comment:
The mapping should work even without this @Mapping, you already writen it above -> uses = LocationMapper.class.

Reply:
I wasn`t sure, but as you said I tested the mapper without the @Mapping annotation and it doesn`t work.